data/service_types_lookup.csv

```diff
@@ -0,0 +1,18 @@
+code,en,fr
```

Comment:
This was created manually by me, I think it'd be much cleaner to have a lookup file from pipeline (json or csv) rather than trying to programatically create one (since it's many-to-many so it has to be assumed to be sorted correctly)

let me know any thoughts